aztk/node_scripts/submit.py

```diff
@@ -113,9 +113,7 @@ def __app_submit_cmd(
     if executor_cores:
         spark_submit_cmd.add_option('--executor-cores', str(executor_cores))
 
-    spark_submit_cmd.add_argument(
-        os.environ['AZ_BATCH_TASK_WORKING_DIR'] + '/' + app + ' ' +
-        ' '.join(['\'' + str(app_arg) + '\'' for app_arg in (app_args or [])]))
+    spark_submit_cmd.add_argument(app + ' ' + ' '.join(['\'' + str(app_arg) + '\'' for app_arg in (app_args or [])]))
```

Comment:
Currently just letting the $AZ_BATCH_TASK_WORKING_DIR be expanded when the spark submit command is executed but could be more explicit and do it here with os.path.expandvars

Reply:
Sure that could be a good idea if we also log that command for debugging purposes